pubsub/rabbitpubsub/rabbit_test.go

```diff
@@ -149,48 +148,6 @@ func (h *harness) Close() {
 
 func (h *harness) MaxBatchSizes() (int, int) { return 0, 0 }
 
-// This test is important for the RabbitMQ driver because the underlying client is
```

Comment:
I don't this test is needed anymore; there are conformance tests and benchmarks that do concurrent sends. We also configure the `Send` batcher with `MaxHandlers=1` so that `SendBatch` isn't called concurrently anyway.